package.json

```diff
@@ -170,6 +171,10 @@
     "*.js.map"
   ],
   "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "default": "./index.js"
+    },
```

Comment:
When I try to build a Vite React project that imports all components from the index file, I get an error from Rollup:
```
RollupError: "GridColumn" is not exported by "node_modules/@vaadin/react-components/index.js", imported by "src/App.tsx".
```

When removing the import and usage of `GridColumn` the build passes.

<details>

<summary>View Code</summary>

```tsx
import {
    Accordion,
    AccordionHeading,
    AccordionPanel,
    AppLayout,
    Avatar,
    AvatarGroup,
    Board,
    BoardRow,
    Button,
    Chart,
    ChartSeries,
    Checkbox,
    CheckboxGroup,
    ComboBox,
    ComboBoxLight,
    ConfirmDialog,
    ContextMenu,
    Map,
    CookieConsent,
    Crud,
    CrudEdit,
    CrudEditColumn,
    CustomField,
    DatePicker,
    DatePickerLight,
    DateTimePicker,
    Details,
    DetailsSummary,
    Dialog,
    DrawerToggle,
    EmailField,
    FormItem,
    FormLayout, Grid, GridColumn,
    GridColumnGroup, GridFilter, GridFilterColumn,
    GridPro,
    GridProEditColumn,
    GridSelectionColumn, GridSortColumn, GridSorter,
    GridTreeColumn, GridTreeToggle,
    HorizontalLayout,
    Icon,
    Iconset,
    IntegerField,
    Item,
    ListBox,
    LoginForm,
    LoginOverlay,
    MenuBar,
    Notification,
    Message,
    MessageInput,
    MessageList,
    MultiSelectComboBox,
    NumberField,
    PasswordField,
    ProgressBar,
    RadioButton,
    RadioGroup,
    RichTextEditor,
    Scroller,
    Select,
    SideNav,
    SideNavItem,
    SplitLayout,
    Tab,
    Tabs,
    TabSheet,
    TextArea,
    TextField,
    TimePicker,
    Tooltip,
    Upload, VerticalLayout, VirtualList
} from "@vaadin/react-components";

function App() {
    return (
        <div>
            <Accordion/>
            <AccordionHeading/>
            <AppLayout/>
            <DrawerToggle/>
            <AccordionPanel/>
            <Avatar/>
            <AvatarGroup/>
            <BoardRow/>
            <Board/>
            <Button/>
            <ChartSeries/>
            <Chart/>
            <CheckboxGroup/>
            <Checkbox/>
            <ComboBox/>
            <ConfirmDialog/>
            <ComboBoxLight/>
            <ContextMenu/>
            <CookieConsent/>
            <CrudEdit/>
            <Crud/>
            <CrudEditColumn/>
            <CustomField/>
            <DatePicker/>
            <DatePickerLight/>
            <DateTimePicker/>
            <DetailsSummary/>
            <Details/>
            <Dialog/>
            <EmailField/>
            <FormItem/>
            <FormLayout/>
            <GridColumnGroup/>
            <GridSelectionColumn/>
            <GridTreeColumn/>
            <GridProEditColumn/>
            <GridPro/>
            <Iconset/>
            <HorizontalLayout/>
            <Icon/>
            <IntegerField/>
            <Item/>
            <ListBox/>
            <LoginForm/>
            <LoginOverlay/>
            <Map/>
            <MenuBar/>
            <MessageInput/>
            <Message/>
            <MessageList/>
            <Notification/>
            <MultiSelectComboBox/>
            <NumberField/>
            <PasswordField/>
            <ProgressBar/>
            <RadioButton/>
            <RadioGroup/>
            <Scroller/>
            <Select/>
            <RichTextEditor/>
            <SplitLayout/>
            <SideNavItem/>
            <SideNav/>
            <Tabs/>
            <Tab/>
            <TabSheet/>
            <TextArea/>
            <TextField/>
            <Tooltip/>
            <TimePicker/>
            <Upload/>
            <VirtualList/>
            <VerticalLayout/>
            <GridColumn/>
            <GridFilter/>
            <GridFilterColumn/>
            <GridSorter/>
            <GridTreeToggle/>
            <GridSortColumn/>
            <Grid/>
        </div>
    )
}

export default App

```

</details>

Reply:
When running in dev mode, it also shows this error in the browser console:
```
Uncaught SyntaxError: ambiguous indirect export: GridColumn
```

For reference, here is the project I'm testing with: 
[vite-demo.zip](https://github.com/vaadin/react-components/files/13960883/vite-demo.zip)

The `package.json` currently points to my local install of the components, you can install your own local version using:
```
npm install /path/to/react-components 
```

Reply:
Looks like the `GridColumn` is exported twice:

```
Conflicting namespaces: "node_modules/@vaadin/react-components/index.js" re-exports "GridColumn" from one of the modules "node_modules/@vaadin/react-components/GridColumn.js" and "node_modules/@vaadin/react-components/Grid.js" (will be ignored).
```

This was previously addressed by https://github.com/vaadin/react-components/pull/89 for individual `Grid` component but it looks like the proper fix would be to remove [re-export in the web component](https://github.com/vaadin/web-components/blob/05618f05f9604d328daa3e6501a5a29d522c2ec4/packages/grid/vaadin-grid.js#L3).